web/src/app/alert/AlertContainer.js

```diff
@@ -67,7 +68,7 @@ class AlertContainer extends Component {
   createQueue() {
     this.alertQueue = [];
 
-    this.isVisible = false;
+    this.isVisible = false; // is the showAlert funcion in progress?
```

Comment:
a variable that needs a comment is a big red flag that a better variable name could be used